prombench/.gitignore

```diff
@@ -1 +1,2 @@
 /prombench
+.env
```

Comment:
This should go inside `.git/info/exclude` this way you exclude env files that are specific to your dev environment.

Reply:
Removed from .gitignore